.expeditor/build-docker-images.sh

```diff
@@ -2,13 +2,24 @@
 set -eu -o pipefail
 
 arch=$1
+
+if [[ $arch == "arm64" ]]; then
```

Comment:
Style. Fill free to skip.

I prefer not doing if/else in this case but to set a default and then override:
```
dockerfile_pkg_version="6"
dockerfile_arch="x86_64"
if [[ $arch == "arm64" ]]; then
    dockerfile_pkg_version="7"
    dockerfile_arch="aarch64"
fi